lib/tarantool_server.py

```diff
@@ -741,6 +741,8 @@ def print_exe(cls):
             color_stdout(' | Found tarantoolctl at {}\n'.format(cls.ctl_path))
         color_stdout('\n' + prefix_each_line(' | ', cls.version()) + '\n',
                      schema='version')
+        color_stdout('Detected build mode: {}\n'.format(
```

Comment:
Could you please add `:` to L737 for consistency?

Reply:
done